platform/ios/INSTALL.md

```diff
@@ -1,6 +1,6 @@
 # Integrating custom builds of the Mapbox iOS SDK into your application
 
-This document explains how to build a development version of Mapbox iOS SDK for use in your own Cocoa Touch application. To use a production-ready version of the SDK, see the [Mapbox iOS SDK homepage](https://mapbox.com/ios-sdk).
+This document explains how to build a development version of Mapbox iOS SDK for use in your own Cocoa Touch application.
```

Comment:
I know we expect people to reach this via a particular flow, but it is also very possible that someone may end up on this page directly. Would it be bad to keep the (redundant with the root INSTALL.md) notice about the pre-built options available at https://www.mapbox.com/install?

Reply:
ohh very good point. I don't think it'd be bad to have a little redundancy to account for a scenario like that.  Will update.